pkg/namespaces/namespaces.go

```diff
@@ -17,7 +17,9 @@ const (
 	nsType        = "ns"
 	podType       = "pod"
 	privateType   = "private"
+	rlkFwdType    = "rootlessport"
```

Comment:
rootlessport -> rootlesskit?

Reply:
this was the example parameter [that @giuseppe used](https://github.com/containers/libpod/issues/5138#issuecomment-618243054).. can change it if you want the parameter to look different